src/doc/guide.md

```diff
@@ -534,21 +531,20 @@ error: aborting due to previous error
 Could not compile `hello_world`.
 ```
 
-Rust will not let us use a value that has not been initialized. Next, let's
-talk about this stuff we've added to `println!`.
+Rust will not let us use a value that has not been initialized. But let us take
+a moment to talk about the new usage of `println!`.
 
-If you include two curly braces (`{}`, some call them moustaches...) in your
-string to print, Rust will interpret this as a request to interpolate some sort
-of value. **String interpolation** is a computer science term that means "stick
-in the middle of a string." We add a comma, and then `x`, to indicate that we
-want `x` to be the value we're interpolating. The comma is used to separate
-arguments we pass to functions and macros, if you're passing more than one.
+If you include two curly braces in your string argument to `println!`, `rustc`
```

Comment:
Aren't programmers expected to know the rest of this? This was just about keeping the section as short and sweet as possible.